### DISCUSSION TITLE
error foundry file not found!!!

### DISCUSSION THREAD

**Discussion by prakharmishra3956** [Cyfrin/foundry-full-course-cu#2638] [Q&A]
![Screenshot (48)](https://github.com/user-attachments/assets/c3fb2d38-ea37-4601-a570-626b8cc3acd0)

**Comment by EngrPips** [comment]
Hello @prakharmishra3956, Please do `forge clean` and then retry. It appears to me that you did everything correctly.

**Comment by BowtiedHarpyEagle** [comment]
Do you realize you have unsaved changes to the files you are working on, on the screenshot? Those white-gray dots next to filename means you have unsaved changes. Maybe it cannot find the file because remappings on the foundry.toml file are not saved yet.

Make a habit to save everything by pressing CTRL-S before you compile or build anything.

**Reply by EngrPips** [reply]
Indeed.
